_sections/events/watm-festival/2020-06-12-1700-connection-practice-meeting.md

```diff
@@ -0,0 +1,20 @@
+---
+type: watm-event
+title: connection-practice-meeting
+name: Connection Practice Meeting
+facilitator: Michelle Kelley
```

Comment:
Is her name Kelly or Kelley?

Reply:
Good catch -- looks like it's "Kelly" in Slack